platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/stats/ReconnectMapMetrics.java

```diff
@@ -125,8 +125,8 @@ public void incrementTransfersFromLearner() {
      */
     @Override
     public void incrementInternalHashes(final int hashNum, final int cleanHashNum) {
-        internalHashes.add(hashNum);
-        internalCleanHashes.add(cleanHashNum);
+        if (hashNum > 0) internalHashes.add(hashNum);
```

Comment:
What's the purpose behind these new conditionals? Wouldn't an input value < 0 indicate a bug?

Reply:
It's to please https://github.com/hashgraph/hedera-services/blob/7c8f5ec4221e23e687f7f9185e0004c5012d9d12/platform-sdk/swirlds-metrics-impl/src/main/java/com/swirlds/metrics/impl/DefaultCounter.java#L63 .

I don't really care or generate <0 values here, but I do pass zeroes. Hence the conditionals.